changelog/unreleased/kong/fix-cjson-t-end.yml

```diff
@@ -0,0 +1,3 @@
+message: |
+  Throw error if T_END found in the middle of JSON `input
```

Comment:
We should make the message more understandable for the uses. `T_END` is not very friendly.

Reply:
fixed